server/routerlicious/packages/routerlicious/src/kafka-service/command.ts

```diff
@@ -23,8 +24,10 @@ export function execute(
         .arguments("<name> <lambda>")
         .action((name: string, lambda: string) => {
             configureLogging(configOrPath);
```

Comment:
Soon, I plan to set Lumberjack up in the `configureLogging` method, and remove the Winston references here - just an FYI. We don't need to do it now